usmqe_tests/conftest.py

```diff
@@ -54,12 +54,14 @@ def logger_testcase(request):
 
 # TODO(fbalak) remove line with `password_confirmation` after
 # https://github.com/Tendrl/api/issues/106 is resolved
+# TODO(fbalak) do not use `"role": "admin"` until issue
+# https://github.com/Tendrl/api/issues/299 is resolved
 @pytest.fixture(
     params=[{
         "name": "Tom Hardy",
         "username": "thardy",
         "email": "thardy@tendrl.org",
-        "role": "admin",
+        "role": "normal",
```

Comment:
I don't think this is valid as a workaround, you have basically redefined what given user is. For *everyone* and *every test case*, which is hardly acceptable (from my point of view).

That said, it could be that nobody from our team cares about this so far, as the details here were still in the motion ...

Reply:
At some point, we didn't even have non admin users in Tendrl, so I would suggest this **instead**:

* create fixture for another admin user account
* create fixture for normal (non admin) user account

and make all test cases to use the admin one (which is the state we have right now) and then update test cases to cover this, and maybe add workaround for the test case which fails because of the issue you just reported.

Reply:
I agree with mbukatov.